### PR TITLE
[stable] tree: promote changes from testing at ad362a929591fc5f1bb97d6fac2572b3ea82ee16

### PR DESCRIPTION
**Pull request by coreosbot-releng**
Triggered by @cverna in https://github.com/coreos/fedora-coreos-streams/issues/504.

### CODE REVIEW

**Comment by cverna** [comment]
Looks like 2 tests are failing

```
[2022-05-23T08:53:28.492Z] May 23 08:53:27 qemu0 systemd[1]: kola-runext-26.service: Failed with result 'exit-code'.
[2022-05-23T08:53:30.998Z] --- FAIL: non-exclusive-test-bucket-0 (26.69s)
[2022-05-23T08:53:30.998Z]     --- FAIL: non-exclusive-test-bucket-0/ext.config.firewall.iptables (0.52s)
[2022-05-23T08:53:30.998Z]             cluster.go:162: Error: Unit kola-runext-26.service exited with code 1
[2022-05-23T08:53:30.998Z]             cluster.go:162: 2022-05-23T08:53:27Z cli: Unit kola-runext-26.service exited with code 1
[2022-05-23T08:53:30.998Z]             harness.go:966: kolet failed: : kolet run-test-unit failed: Process exited with status 1
[2022-05-23T08:54:09.630Z] systemctl status kola-runext.service:
[2022-05-23T08:54:09.630Z] × kola-runext.service
[2022-05-23T08:54:09.630Z]      Loaded: loaded (/etc/systemd/system/kola-runext.service; static)
[2022-05-23T08:54:09.630Z]      Active: failed (Result: exit-code) since Mon 2022-05-23 08:54:06 UTC; 99ms ago
[2022-05-23T08:54:09.630Z]     Process: 1485 ExecStart=/usr/local/bin/kola-runext-test.sh (code=exited, status=1/FAILURE)
[2022-05-23T08:54:09.630Z]    Main PID: 1485 (code=exited, status=1/FAILURE)
[2022-05-23T08:54:09.630Z]         CPU: 3ms
[2022-05-23T08:54:09.630Z] 
[2022-05-23T08:54:09.630Z] May 23 08:54:06 qemu0 systemd[1]: Started kola-runext.service.
[2022-05-23T08:54:09.630Z] May 23 08:54:06 qemu0 kola-runext-test.sh[1485]: + . /var/opt/kola/extdata/commonlib.sh
[2022-05-23T08:54:09.635Z] May 23 08:54:06 qemu0 kola-runext-test.sh[1485]: + case "${AUTOPKGTEST_REBOOT_MARK:-}" in
[2022-05-23T08:54:09.635Z] May 23 08:54:06 qemu0 kola-runext-test.sh[1486]: ++ find /var/crash -type f -name vmcore
[2022-05-23T08:54:09.635Z] May 23 08:54:06 qemu0 kola-runext-test.sh[1485]: + kcore=
[2022-05-23T08:54:09.635Z] May 23 08:54:06 qemu0 kola-runext-test.sh[1485]: + test -z ''
[2022-05-23T08:54:09.635Z] May 23 08:54:06 qemu0 kola-runext-test.sh[1485]: + fatal 'No kcore found in /var/crash'
[2022-05-23T08:54:09.635Z] May 23 08:54:06 qemu0 kola-runext-test.sh[1485]: + echo 'No kcore found in /var/crash'
[2022-05-23T08:54:09.635Z] May 23 08:54:06 qemu0 kola-runext-test.sh[1485]: No kcore found in /var/crash
[2022-05-23T08:54:09.635Z] May 23 08:54:06 qemu0 kola-runext-test.sh[1485]: + exit 1
[2022-05-23T08:54:09.635Z] May 23 08:54:06 qemu0 systemd[1]: kola-runext.service: Main process exited, code=exited, status=1/FAILURE
[2022-05-23T08:54:09.635Z] May 23 08:54:06 qemu0 systemd[1]: kola-runext.service: Failed with result 'exit-code'.
[2022-05-23T08:54:09.635Z] --- FAIL: ext.config.kdump.crash (65.68s)
[2022-05-23T08:54:09.635Z]         cluster.go:162: Error: Unit kola-runext.service exited with code 1
[2022-05-23T08:54:09.635Z]         cluster.go:162: 2022-05-23T08:54:06Z cli: Unit kola-runext.service exited with code 1
[2022-05-23T08:54:09.635Z]         harness.go:966: kolet failed: : kolet run-test-unit failed: Process exited with status 1
[2022-05-23T08:54:09.635Z] FAIL, output in /srv/fcos/tmp/fcosKola-I0X97/kola/rerun
[2022-05-23T08:54:09.635Z] Error: harness: test suite failed
[2022-05-23T08:54:09.635Z] 2022-05-23T08:54:09Z cli: harness: test suite failed
```

**Comment by dustymabe** [comment]
@cverna - this one needs an extra commit on top to switch us over to Fedora 36.. You can `git cherry-pick -x aee45b40c041e45e63553e142d375ea730135561` to pick it up.

File: manifest.yaml [hunk 1]
```diff
@@ -2,6 +2,6 @@ variables:
   stream: stable
   prod: true
 
-releasever: 35
+releasever: 36
```
Comment (cverna) [inline manifest.yaml:5]: This is expected

**Comment by cverna** [comment]
So it looks like the kdump test is still failing 
```
[2022-05-24T07:16:14.468Z] === RUN   ext.config.kdump.crash
[2022-05-24T07:17:22.072Z] systemctl status kola-runext.service:
[2022-05-24T07:17:22.072Z] × kola-runext.service
[2022-05-24T07:17:22.072Z]      Loaded: loaded (/etc/systemd/system/kola-runext.service; static)
[2022-05-24T07:17:22.072Z]      Active: failed (Result: exit-code) since Tue 2022-05-24 07:17:15 UTC; 107ms ago
[2022-05-24T07:17:22.072Z]     Process: 1475 ExecStart=/usr/local/bin/kola-runext-test.sh (code=exited, status=1/FAILURE)
[2022-05-24T07:17:22.072Z]    Main PID: 1475 (code=exited, status=1/FAILURE)
[2022-05-24T07:17:22.072Z]         CPU: 3ms
[2022-05-24T07:17:22.072Z] 
[2022-05-24T07:17:22.072Z] May 24 07:17:15 qemu0 systemd[1]: Started kola-runext.service.
[2022-05-24T07:17:22.072Z] May 24 07:17:15 qemu0 kola-runext-test.sh[1475]: + . /var/opt/kola/extdata/commonlib.sh
[2022-05-24T07:17:22.072Z] May 24 07:17:15 qemu0 kola-runext-test.sh[1475]: + case "${AUTOPKGTEST_REBOOT_MARK:-}" in
[2022-05-24T07:17:22.072Z] May 24 07:17:15 qemu0 kola-runext-test.sh[1476]: ++ find /var/crash -type f -name vmcore
[2022-05-24T07:17:22.072Z] May 24 07:17:15 qemu0 kola-runext-test.sh[1475]: + kcore=
[2022-05-24T07:17:22.072Z] May 24 07:17:15 qemu0 kola-runext-test.sh[1475]: + test -z ''
[2022-05-24T07:17:22.072Z] May 24 07:17:15 qemu0 kola-runext-test.sh[1475]: + fatal 'No kcore found in /var/crash'
[2022-05-24T07:17:22.072Z] May 24 07:17:15 qemu0 kola-runext-test.sh[1475]: + echo 'No kcore found in /var/crash'
[2022-05-24T07:17:22.072Z] May 24 07:17:15 qemu0 kola-runext-test.sh[1475]: No kcore found in /var/crash
[2022-05-24T07:17:22.072Z] May 24 07:17:15 qemu0 kola-runext-test.sh[1475]: + exit 1
[2022-05-24T07:17:22.072Z] May 24 07:17:15 qemu0 systemd[1]: kola-runext.service: Main process exited, code=exited, status=1/FAILURE
[2022-05-24T07:17:22.072Z] May 24 07:17:15 qemu0 systemd[1]: kola-runext.service: Failed with result 'exit-code'.
[2022-05-24T07:17:22.072Z] --- FAIL: ext.config.kdump.crash (65.00s)
[2022-05-24T07:17:22.072Z]         cluster.go:162: Error: Unit kola-runext.service exited with code 1
[2022-05-24T07:17:22.072Z]         cluster.go:162: 2022-05-24T07:17:15Z cli: Unit kola-runext.service exited with code 1
[2022-05-24T07:17:22.072Z]         harness.go:966: kolet failed: : kolet run-test-unit failed: Process exited with status 1
[2022-05-24T07:17:22.072Z] FAIL, output in /srv/fcos/tmp/fcosKola-ILF3O/kola/rerun
[2022-05-24T07:17:22.072Z] Error: harness: test suite failed
[2022-05-24T07:17:22.072Z] 2022-05-24T07:17:18Z cli: harness: test suite failed
```

**Comment by cverna** [comment]
Seems related to https://github.com/coreos/fedora-coreos-tracker/issues/1079  but as far as I can tell we should have the fix in that PR.

**Comment by cverna** [comment]
Running kola locally also fails --> https://paste.centos.org/view/raw/8224af4e
But just running the `ext.config.kdump.crash` test actually works --> https://paste.centos.org/view/raw/8acc3c48

**Comment by dustymabe** [comment]
I cherry picked 7385403 and pushed here. That should get us past this failure.

**Review by dustymabe — APPROVED** [review]
LGTM

**Pull request merged**